backend/pkg/dbschemas/sql/postgresql/queries/system.sql

```diff
@@ -256,12 +256,12 @@ ORDER BY
 SELECT
     con.conname AS constraint_name,
     con.contype::TEXT AS constraint_type,
-    con.connamespace::regnamespace::TEXT AS schema_name,
+    nsp.nspname AS schema_name,
```

Comment:
Using the table from nsp.nspname as the `con.connamespace::regnamespace::TEXT` added an extra quote that is only needed in some cases and we mostly want to work with unescaped values.